docs/enrollment-pathways/Low-income.md

```diff
@@ -0,0 +1,75 @@
+
+## **Alternative Flows**
+
+- Suppose the transit rider does not have a desktop computer. In this case, they open the web application at [benefits.calitp.org](benefits.calitp.org) in a mobile browser on their iOS or Android tablet or mobile device to complete enrollment using the basic flow.
+
+- Suppose the transit rider cannot authenticate with [Login.gov](Login.gov), or will not create an account. In either case, the app cannot determine their Calfresh Program participation status and they cannot enroll their contactless debit or credit card for a reduced fare.
+
+- Suppose the CDT Identity Gateway returns a status of FALSE for Calfresh Program participation status. In that case, the Cal-ITP Benefits app will not allow the transit rider to enroll their contactless debit or credit card for a reduced fare.
+
+- Suppose the debit or credit card expires or is canceled by the issuer. In that case, the transit rider must repeat the basic flow to register a new debit or credit card.
+
+- When the initial transit benefit enrollment period ends after one year from the date of enrollment, the transit rider must repeat the basic flow to re-enroll.
+
+- Suppose the transit rider attempts to re-enroll for a transit benefit as a CalFresh Program participant three months after their enrollment period started. The app will inform them they must wait re-enroll within 14 days of the benefit expiration.
+
+- Suppose the transit rider doesn’t re-enroll for a transit benefit after one year, but continues paying for transit using the card they registered. The transit operator will charge the rider full fare.
+
+- If the transit rider uses more than one debit or credit card to pay for transit, they repeat the basic flow for each card.
+
+## **Postcondition**
+
+The transit rider receives a fare reduction each time they use the debit or credit card they registered to pay for transit rides. The number of times they can use the card to pay for transit is unlimited, but the benefit expires one year after enrollment.
+
+## **Benefits**
+
+- The transit rider no longer needs cash to pay for transit rides.
+
+- The transit rider doesn’t have to lock up funds on a closed-loop card offered by the transit agency.
+
+- The transit rider pays for transit rides with their debit or credit card, just as they do for groceries, a cup of coffee, or any other good or service.
+
+- The transit rider can enroll in a transit benefit from home when convenient; they do not have to visit a transit agency in person.
+
+- Secure state and federal solutions manage the transit rider’s personal identifiable information (PII): [Login.gov](Login.gov) and the California Department of Technology Identity Gateway (IdG). Transit riders do not have to share personal information with local transit operators.
+
+- Benefit enrollment takes minutes rather than days or weeks.
+
+- Benefit enrollment doesn’t require online accounts with private companies.
```

Comment:
These benefits (for Benefits, hah) are the same for all eligibility types. But I guess it's okay to repeat them.

One benefit that is SPECIFIC to low-income is that by using CalFresh status, the rider does not have to go through another income verification process, and the transit agency does not have to create an income verification process that they have to then manage and get approved by their Board of Directors and so on. And what if every transit agency had a different definition of low income? This allows for potentially having the _same_ definition of "low income" across the state.